### ISSUE TITLE
Tag a new release

### ISSUE THREAD

**Issue by blkperl** [opencontainers/umoci#478]
Hey @cyphar could you tag a new release please?

**Comment by cyphar** [comment]
Yeah, it's probably about time to do a new release. I want to fix up a few minor things (such as how test coverage is done) and then I'll do a release.

**Comment by nealef** [comment]
Any update? Anything you'd like done that we can help with?

**Comment by cyphar** [comment]
I need to finish updating umoci to image-spec 1.1, there are only a handful of things to do but I've been busy with some large runc rewrites (and I'm on vacation for another week). I'll make sure to take a look when I get home next week -- sorry about that...

**Comment by nealef** [comment]
No worries. I have added a cache subcommand that will create an image cache for containerd to use. It's very single-purposed so I'm not sure it would be of general interest. It requires a snapshotter and content plugin to containerd to exploit so it's definitely a niche case.

**Comment by jeffb4** [comment]
Will the new release automatically generate `linux/arm64` binaries?

**Comment by cyphar** [comment]
No, in the past I only ever built `linux/amd64`. I'll add some extra binaries to `make release`.
